package.json

```diff
@@ -183,6 +183,7 @@
     "getos": "^3.2.1",
     "got": "^14.2.1",
     "hpagent": "^1.2.0",
+    "is-elevated": "^4.0.0",
```

Comment:
I am not a big fan of this library as it has no support, not been updated since Aug 11, 2021, it fetched two other library the `is-admin` and `is-root` which have the same problem.

The `is-admin` is running the following on windows `fsutil dirty query C:` and the `is-root` is simply checking the user uid..

I don't know what is our policy for adding packages cc @benoitf

Reply:
I would not proceed with this library

Reply:
I mean, the library is OK from maintenance POV as they're managed by sindresorhus so there is support

but I'm not sure why we would need that library, I mean, in which case people run Podman Desktop with admin privileges ?

I would not allow ports < 1024 in all cases (Windows, macOS or Linux)

Reply:
>I would not allow ports < 1024 in all cases (Windows, macOS or Linux)

This simplifies the whole checks, will going to update the PR by removing this check.

I just rely on that scenario, that all ports, that are below 1024 by default requires administrative permissions to expose.

Reply:
@vzhukovs yes but I think it's simpler/easier/more robust to not allow that anyway using Podman Desktop

thanks for updating the PR

Reply:
@benoitf new changes are in place